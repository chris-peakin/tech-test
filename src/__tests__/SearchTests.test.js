import renderer from "react-test-renderer";
import Search from "../components/SearchComponent";

describe("search component", () =>{
    const validProps = "generic search term";
    const testRenderer = renderer.create(<Search setSearchResults = {validProps} />);

    it("search component renders correctly", () =>{
        expect(testRenderer).toMatchSnapshot();
    });
});