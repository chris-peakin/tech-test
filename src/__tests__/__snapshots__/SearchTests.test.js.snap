// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`search component search component renders correctly 1`] = `
<div
  className="Search"
>
  <form
    className="searchForm"
    onSubmit={[Function]}
  >
    <input
      className="search-input"
      onChange={[Function]}
      type="text"
    />
    <button
      className="search-button"
      type="submit"
    >
      Tally Ho!
    </button>
  </form>
</div>
`;
